input-explainer.md

```diff
@@ -228,21 +228,35 @@ For `tracked-pointer` input sources, it is often appropriate for the application
 #### Choosing renderable models
 The majority of `tracked-pointer` input sources will have a non-null `gamepad` attribute on the `XRInputSource` object. The `Gamepad`'s `id` is used to determine what should be rendered if the app intends to visualize the input source itself, rather than an alternative virtual object. (See the section on [Button and Axis State](#button-and-axis-state) for more details.)
 
-The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the `Gamepad`'s `id` must be used as a key to load an appropriate resources from the application's server or a CDN. The example below presumes that the `getInputSourceRenderableModel` call would do the required lookup and caching.
+The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the `XRInputSource`'s `profiles` must be used to identify and load an appropriate resources. (For example, from the application's server, a CDN, or a local cache.) The `profiles` provides a list of strings that identify the device, given in descending order of preference.
+
+For example, the Samsung Odyssey controller is a variant of the standard Windows Mixed Reality controller. As a result, the `profiles` for a that controller could be:
 
 ```js
-function loadRenderableInputModels(xrInputSource) {
-  // Don't load renderable models if the input source doesn't have a gamepad. 
-  if (!inputSource.gamepad)
-    return;
+// Exact strings are examples only.
+["samsung-odyssey", "windows-mixed-reality"]
+```
 
-  // Retrieve a mesh to render based on the gamepad object's id and handedness
-  let renderableModel = getInputSourceRenderableModel(inputSource.gamepad.id, inputSource.handedness);
-  if (!renderableModel)
-    return;
+Applications should iterate through the list until a string is located that corresponds to a known model, which should then be used when rendering the input device. The example below presumes that the `getInputSourceRenderableModel` call would do the required lookup and caching.
```

Comment:
>Applications should iterate through the list until a string is located that corresponds to a known model, which should then be used when rendering the input device.

Can we modify this to also mention that developers are expected to know what to do with the layout implied by the profile if they consider if recognized?  Perhaps calling out that it's important to animate all controller components even if the experience doesn't use the data they provide.